recipes/mdacli/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "mdacli" %}
+{% set version = "0.1.12" %}
+{% set sha256 = "a26ec097677937b66aab3330b17a6360aa96b6760295ae44792639158c35175d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [python_impl == 'pypy']
+  skip: true  # [py<39]
+  skip: true  # [py>39]
```

Don't skip PyPy by default, don't merge the PyPy migration PR later.
```suggestion
  skip: true  # [py<39]
  skip: true  # [py>39]
```